tools/demo/src/filters/color-overlay.js

```diff
@@ -2,10 +2,11 @@ export default function ()
 {
     this.addFilter('ColorOverlayFilter', {
         fishOnly: true,
-        args: [0xff0000],
+        args: [0xff0000, 0.5],
```

Comment:
preference to default to `1` not `0.5` in the demo

Reply:
set it to `1`